esx_service/utils/vmdk_utils.py

```diff
@@ -232,7 +232,7 @@ def list_vmdks(path, volname="", show_snapshots=False):
 def vmdk_is_a_descriptor(path, file_name):
     """
     Is the file a vmdk descriptor file?  We assume any file that ends in .vmdk,
-    does not have -delta or -flat or he likes at the end of filename, 
+    does not have -delta or -flat or he likes at the end of filename,
```

Comment:
@govint We only disallow space in tenant name and volume name during creation. So with this fix, issue #892 can still happen. We cannot control the datastore name. In VSphere, spaces in datastore name are allowed.